src/compiler/parser.ts

```diff
@@ -345,7 +345,7 @@ module ts {
     module Parser {
         // Share a single scanner across all calls to parse a source file.  This helps speed things
         // up by avoiding the cost of creating/compiling scanners over and over again.
-        const scanner = createScanner(ScriptTarget.Latest, /*skipTrivia:*/ true);

```

Comment:
It is inconsistent throughout our code base, but the majority of comments have no colon. For example, there are none in checker.ts, our largest file. I chose to align with that. My preference is no colon as I don't think it adds any value.